Source/ROLib/Modules/ModuelROPayload.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace ROLib
+{
+    public class ModuleROPayload : ModuleDeployableSolarPanel
+    {
+        protected PartModule pmROTank;
+
+        public override void OnStart(StartState state)
+        {
+            base.OnStart(state);
+            pmROTank = part.Modules["ModuleROTank"];
```

Comment:
Use `nameof`. Or better yet - some generic method for fetching modules, like `Modules.GetModule<T>`.

Reply:
Fixed.  Will reopen PR to avoid an accidental merge commit.